package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mics",
-  "version": "0.5.0",
+  "version": "0.6.1",
```

Comment:
I think it's best to just ignore versions in the feature branches. Once the feature branch is merged, we cut a new version directly from master. This prevents the feature branch having to guess at the version, or getting conflicts because the version changed from another feature being merged.